test/tests/functional/pbs_snapshot_unittest.py

```diff
@@ -148,26 +150,26 @@ def setup_queues_nodes(self, num_partitions):
 
         return (queues, nodes)
 
-    def take_snapshot(self, parent_dir, acct_logs=None, daemon_logs=None,
-                      obfuscate=None):
+    def take_snapshot(self, acct_logs=None, daemon_logs=None,
+                      obfuscate=None, with_sudo=True):
```

Comment:
I've noticed that the sudo parameter has been added to several functions while others use self.sudo. Now you're introducing another parameter but naming it differently. It seems rather inconsistent.

Reply:
Sorry for the confusion, I've renamed the function arguments to be more consistent, please let me know if things make more sense now. Thanks.